bin/linters.sh

```diff
@@ -24,7 +24,7 @@ docker run\
   -w /go/src/istio.io/istio\
   gcr.io/mukai-istio/linter:bbcfb47f85643d4f5a7b1c092280d33ffd214c10\
   --config=./lintconfig.json \
-  ./... || true
```

Comment:
woohoo!!

If anything is causing the linter to fail, let's make it part of the exclusion and create p0 bug against the owner. The owner of the bug can reproduce the issues by remove the directory name from the exclusion list.